src/main/java/network/brightspots/rcv/ContestConfig.java

```diff
@@ -947,6 +962,12 @@ boolean isContinueUntilTwoCandidatesRemainEnabled() {
     return rawConfig.rules.continueUntilTwoCandidatesRemain;
   }
 
+  Integer getStopTabulationAtRound() {
```

Comment:
Nit: `getStopTabulationEarlyOnRound`

Reply:
This still says "at" instead of "on", so recommend changing it to be consistent with everything else.